Dockerfile

```diff
@@ -1,25 +1,41 @@
-FROM node:8.2.1-alpine
+FROM node:8.9
```

Comment:
All of Home Assistant infrastructure runs on Alpine and it runs great. I don't see any reason to change that.

Reply:
... and my question is, **why not?** 

I checked home-assistant and it start from ["python:3.6"](https://github.com/home-assistant/home-assistant/blob/dev/Dockerfile#L5) that start from ["buildpack-deps:jessie"](https://github.com/docker-library/python/blob/a1aa406bfd8c7b129e6e0ee0ba972b863624ac0d/3.6/jessie/Dockerfile#L7) , so why use alpine on home-assistant-polymer?

The concept behind Alpine was to create a light and simple firewall/proxy distro that could run in RAM... it's very useful for "production docker" but has less sense in a development environment.
This environment docker serves only to develop and create the frontend package, it not provide service for the production. This is so true that the first step that you do is to [add/install the build and versioning files](https://github.com/home-assistant/home-assistant-polymer/blob/master/Dockerfile#L7)

Lastly, I have shown that it does not work ( Issue #934 ) and I have not found any automatic system that validates the current compilation via docker (if that is not the case please tell me so. I have searched on travis but no dockers are used, not even I have not seen web-hooks pointing on docker hub)

**Why I choose node:8.9 and not node:8.2.1?**
The home-assistant-polymer (as user of node), why should it be frozen even at patch version? the idea is to use official LTS provided by NodeJS and do not loose all the security and optimization patches ( and finally it start from "buildpack-deps:jessie" the same of "python:3.6" on "home-assistant" )

Reply:
@Odyno You can still use `-alpine` though https://github.com/nodejs/docker-node/blob/8498ff5302c19506c9edc3ce152f2bd4aa6b26b3/8/alpine/Dockerfile
`gcc, make` etc. could be easily pruned. and python3 being up-to-date looks not so relevant here.

Other changes would be nice to have, easily get compiled frontend using just docker.